xbmc/network/upnp/UPnPInternal.cpp

```diff
@@ -38,6 +38,28 @@ using namespace XFILE;
 
 namespace UPNP
 {
+// add some hard coded allowed subtitle just like
```

Comment:
please drop the history context from the comment. There's no need for it and also feels wrong to link to github commit addresses urls (we can always move elsewhere...) or past history log lines. You can extend the commit message with that context if you want so that it keeps part of the commit history.

The block regarding the iana mimetypes and UPnP specs is appreciated though, and can stay.